### DISCUSSION TITLE
Error in setting up the relationships between tables - PostgreSQL DB

### DISCUSSION THREAD

**Discussion by alexandrakubatieva** [Budibase/budibase#12035] [Help]
Hi, I've created a PostgreSQL data source and managed to connect it to my database, but I'm getting an error when trying to select **Relationships** tab - I see an empty screen without **Define Relationship** option. A console log might be related:
![image](https://github.com/Budibase/budibase/assets/111291379/31ff9bc9-151e-4bf5-95ea-8b4f36876db9)
The app isn't self-hosted.

**Comment by ConorWebb96** [comment]
Hey @alexandrakubatieva,

- Are you self-host or cloud?
- What version of Budibase are you on?
- What is inside your tables tab?
- Have you fetched your tables? 
- Can you provide some screenshots of the tables tables tab? As well as the modal for fetching.
- Have your tables synced into Budibase? For example do you have a data source called postgreSQL with tables contained within on the left side of your screen?

Thanks for information in advance.
